library/src/webapp/js/portal/portal.sidebar.collapse.js

```diff
@@ -29,11 +29,15 @@ class SidebarCollapseButton {
   }
 
   async setCollapsed(collapsed) {
+    if (!portal.user || !portal.user.id) {
+        return; // Exit the function early if the user is not logged in.
```

Comment:
try and stick to two spaces in new javascript. 2 spaces is a lot better when you write functional code. functional code has a lot of nesting, potentially.